libs/application/templates/financial-aid/src/lib/messages/status.ts

```diff
@@ -0,0 +1,210 @@
+    description: 'Action title of the card',
+  },
+})
+
+export const header = defineMessages({
+  new: {
+    id: 'fa.application:section.header.new',
+    defaultMessage: 'Umsókn móttekin',
+    description: 'New text',
+  },
+  inProgress: {
+    id: 'fa.application:section.header.inProgress#markup',
+    defaultMessage: 'Umsókn í vinnslu til útgreiðslu í {month} {year}',
+    description: 'In progress text',
+  },
+  approved: {
+    id: 'fa.application:section.header.approved',
+    defaultMessage: 'Umsókn samþykkt',
+    description: 'Approved text',
+  },
+  rejected: {
+    id: 'fa.application:section.header.rejected',
+    defaultMessage: 'Umsókn synjað',
+    description: 'Rejected text',
+  },
+})
+
+export const rejectionMessage = defineMessages({
+  explanation: {
+    id: 'fa.application:section.rejectionMessage.explanation#markup',
+    defaultMessage:
+      'Umsókn þinni um fjárhagsaðstoð í ágúst hefur verið synjað {rejectionComment}.',
+    description: 'Explanation of rejection',
+  },
+  explanationLink: {
+    id: 'fa.application:section.rejectionMessage.explanationLink',
+    defaultMessage:
+      'Smelltu á hlekkinn hér fyrir neðan til að kynna þér reglur um fjárhagsaðstoð.',
+    description: 'Text to direct to the rules link',
+  },
+  rulesPageLink: {
+    id: 'fa.application:section.rejectionMessage.rulesPageLink',
+    defaultMessage: 'Sjá reglur um fjárhagsaðstoð',
+    description: 'Text on link to rules page',
+  },
+  appealTitle: {
+    id: 'fa.application:section.rejectionMessage.appealTitle',
+    defaultMessage: 'Málskot',
+    description: 'Appeal title',
+  },
+  appealDescription: {
+    id: 'fa.application:section.rejectionMessage.appealDescription#markup',
+    defaultMessage:
+      'Bent skal á að unnt er að skjóta ákvörðun þessari til áfrýjunarnefndar þíns sveitarfélags. Skal það gert skriflega og innan fjögurra vikna. Fyrir frekari upplýsingar um málskot hafðu samband með tölvupósti á netfangið [{email}]({email}). \n\n Ákvörðun ráðsins má síðan skjóta til úrskurðarnefndar velferðarmála, Katrínartúni 2, 105 Reykjavík innan þriggja mánaða.',
+    description: 'Appeal title',
+  },
+})
+
+export const spouseAlert = defineMessages({
+  title: {
+    id: 'fa.application:section.spouseAlert.title',
+    defaultMessage: 'Gögn maka vantar',
+    description: 'Title of alert',
+  },
+  message: {
+    id: 'fa.application:section.spouseAlert.message',
+    defaultMessage:
+      'Umsóknin verður tekin til úrvinnslu þegar maki skilar inn gögnum. Maki hefur fengið tölvupóst þess efnis.',
+    description: 'Message of alert',
+  },
+})
+
+export const spouseApproved = defineMessages({
+  message: {
+    id: 'fa.application:section.spouseApproved.message#markup',
+    defaultMessage:
+      'Umsóknin maka þíns um fjárhagsaðstoð í {month} er samþykkt. Maki þinn fær frekari upplýsingar um veitta aðstoð.',
+    description: 'Message for spouse when application is apporoved',
+  },
+})
```

Comment:
Most of the strings that are on the spouseStatus page are the same as on the applicantStatus page, so I feel like it is fine to have it all in the same place. It might be a bit confusing if few of the strings for the spouseStatus page are in one file and others in another file.